README.rst

```diff
@@ -20,10 +20,11 @@ number-parser requires Python 3.6+.
 Usage
 =====
 
-The library provides three major APIs which corresponds to the following common usages.
+The library provides the following common usages.
 
-Interface #1: Multiple numbers
-------------------------------
+--------
+- **Converting numbers in-place**
+--------
```

Comment:
I think only the text should change, the formatting should remain the same, in line with the rest of the file:

```suggestion
Converting numbers in-place
---------------------------
```

The same applies to the other headers.

Reply:
Sorry for that, I have changed the formatting so that it will be in line with the rest of the file.